srv/src/main/java/my/bookshop/swagger/SwaggerSecurityConfig.java

```diff
@@ -0,0 +1,19 @@
+package my.bookshop.swagger;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+@EnableWebSecurity
+@Order(1)
+public class SwaggerSecurityConfig extends WebSecurityConfigurerAdapter {
+
+	@Override
+	protected void configure(HttpSecurity http) throws Exception {
+		http.requestMatchers().antMatchers("/swagger/**").and().authorizeRequests().anyRequest().permitAll();
```

Comment:
Is this code required to enforce authentication for the swagger UI content? Or should we allow anonymous access to the swagger UI?

Reply:
This code enables anonymous access to the swagger UI. This is required as Spring Security (and also our Security configs) by default requires authentication for all endpoints once enabled.

Reply:
ok, then it looks good to me.